File: examples/src/Shapes/EmbeddedVideoFrame.cs
using System.IO

import aspose.slides as slides
using Aspose.slides.Export

namespace Aspose.slides.Examples.CSharp.shapes 
{
    public class EmbeddedVideoFrame
    {
        public static void Run()
        {
            #ExStart:EmbeddedVideoFrame
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Shapes()
            videoDir = RunExamples.GetDataDir_Video()
            resultPath = Path.Combine(RunExamples.OutPath, "VideoFrame_out.pptx")

            # Create directory if it is not already present.
            bool IsExists = System.IO.Directory.Exists(dataDir)
            if (!IsExists)
                System.IO.Directory.CreateDirectory(dataDir)
            # Instantiate Presentation class that represents the PPTX
            with slides.Presentation() as pres:
            {

                # Get the first slide
                sld = pres.slides[0]

                # Embedd vide inside presentation
                IVideo vid = pres.Videos.AddVideo(new FileStream(videoDir + "Wildlife.mp4", FileMode.Open), LoadingStreamBehavior.ReadStreamAndRelease)

                # Add Video Frame
                IVideoFrame vf = sld.shapes.AddVideoFrame(50, 150, 300, 350, vid)

                # Set video to Video Frame
                vf.EmbeddedVideo = vid

                # Set Play Mode and Volume of the Video
                vf.PlayMode = VideoPlayModePreset.Auto
                vf.Volume = AudioVolumeMode.Loud

                # Write the PPTX file to disk
                pres.save(resultPath, slides.export.SaveFormat.PPTX)
            }
            #ExEnd:EmbeddedVideoFrame
        }
    }
}

File: examples/src/shapes/embedded_video_frame.py
import aspose.slides as slides


def embedded_video_frame(global_opts):
    # Instantiate Presentation class that represents the PPTX
    with slides.Presentation() as pres:
        # Get the first slide
        slide = pres.slides[0]

        # Embed video inside presentation
        with open(global_opts.data_dir + "video.mp4", "rb") as in_file:
            video = pres.videos.add_video(in_file, slides.LoadingStreamBehavior.READ_STREAM_AND_RELEASE)

            # Add Video Frame
            vf = slide.shapes.add_video_frame(50, 150, 300, 350, video)

            # Set video to Video Frame
            vf.embedded_video = video

            # Set Play Mode and Volume of the Video
            vf.play_mode = slides.VideoPlayModePreset.AUTO
            vf.volume = slides.AudioVolumeMode.LOUD

        # Write the PPTX file to disk
        pres.save(global_opts.out_dir + "shapes_embed_video_frame_out.pptx", slides.export.SaveFormat.PPTX)
